SCons/Platform/posix.py

```diff
@@ -62,9 +62,15 @@ def escape(arg):
     return '"' + arg + '"'
 
 
-def exec_subprocess(l, env):
-    proc = subprocess.Popen(l, env = env, close_fds = True)
-    return proc.wait()
+if process_spawner:
+    from SCons.Job import spawner_tls
+
+    def exec_subprocess(l, env):
```

Comment:
I modified only `exec_subprocess`, and not `exec_popen3` / `_subproc` because they are more complex to wrap (although not impossible), and the "big win" for performance is from this function, which is the most commonly called (AFAIK)